### ISSUE TITLE
BigWigs causing frame stutter in Underrot with Priest

### ISSUE THREAD

**Issue by stigmatiic** [BigWigsMods/BigWigs#613]
### What steps will reproduce the problem?  
1.  Running BigWigs with no customizations
2.  Entering Underrot with a Priest (specifically Shadow, but happens on Disc too)
3.  Start attacking the fetid maggot and you'll see a stutter

### What version of BigWigs are you using? (Stating 'latest' is not useful)
v136.1

### Do you have an error log of what happened?
There is no error log

### Any additional information? (example: WoW language if not English) (Attach screenshots here if available, do not link externally)

**Comment by funkydude** [comment]
You think this is due to BigWigs, because..?
Ran this hundreds of times without issues.

**Comment by stigmatiic** [comment]
I went through each addon and disabled them for testing. Disabling BigWigs was the addon that made the stuttering stop. Running my other addons (GTFO, ElvUI, WA2) with BigWigs turned off did not produce the stutter. This wasn’t an issue until it popped up last night - never was a problem before. I’ve also removed my interface folder and copied the adding back in, but the issue remained.

Thanks,
Josh

On Feb 25, 2019, at 10:27 AM, funkydude <notifications@github.com<mailto:notifications@github.com>> wrote:


You think this is due to BigWigs, because..?
Ran this hundreds of times without issues.
